ui/apps/platform/src/Containers/Integrations/utils/tableColumnDescriptor.ts

```diff
@@ -293,8 +296,20 @@ const tableColumnDescriptor: Readonly<IntegrationTableColumnDescriptorMap> = {
         ],
         azure: [
             { accessor: 'name', Header: 'Name' },
-            { accessor: 'docker.endpoint', Header: 'Endpoint' },
-            { accessor: 'docker.username', Header: 'Username' },
+            {
+                accessor: (integration) =>
+                    'azure' in integration
+                        ? String(integration?.azure?.endpoint)
+                        : String(integration?.docker?.endpoint),
```

Comment:
I was going to suggest removing the safe access operator `?.` here, since TypeScript should correctly infer what properties are available based on the `in` check above, but I see that the integration section makes heavy use of inferred `any`. This would be nice to update in the future but certainly way out of scope for now.